src/EnergyPlus/DataZoneEquipment.cc

```diff
@@ -757,12 +743,12 @@ void GetZoneEquipmentData(EnergyPlusData &state)
     // Allocate TermUnitSizing array and set zone number
     if (locTermUnitSizingCounter > 0) {
         state.dataSize->NumAirTerminalUnits = locTermUnitSizingCounter;
-        TermUnitSizing.allocate(state.dataSize->NumAirTerminalUnits);
+        state.dataSize->TermUnitSizing.allocate(state.dataSize->NumAirTerminalUnits);
         for (int loopZoneNum = 1; loopZoneNum <= state.dataGlobal->NumOfZones; ++loopZoneNum) {
             {
-                auto &thisZoneEqConfig(state.dataZoneEquip->ZoneEquipConfig(loopZoneNum));
+                auto &thisZoneEqConfig = state.dataZoneEquip->ZoneEquipConfig(loopZoneNum);
```

Comment:
@amirroth I was changing this notation and noticed hyour comment this morning, constructor notation for scalars only. Revert change?

Reply:
1. A reference is a scalar.
2. I made a similar comment about references later.
3. Leave it, thx!!